release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1239,6 +1239,9 @@ With this release, replacement control plane nodes are assigned to the correct i
 
 * Previously, the Kubernetes scheduler could skip scheduling certain pods for a node that received multiple restart operations. {product-title} {product-version} counteracts this issue by including the `KubePodNotScheduled` alert for pods that cannot be scheduled within 30 minutes. (link:https://issues.redhat.com/browse/OCPBUGS-2260[*OCPBUGS-2260*])
 
+* Previously, if more than one extra label was defined for Thanos Ruler, then the statefulset could enter a recreation loop because prometheus-operator would not add the labels in a specified order each time it reconciled the custom resource. After this fix, prometheus-operator now sorts extra labels before adding them to the statefulset. (link:https://issues.redhat.com/browse/OCPBUGS-6055[*OCPBUGS-6055*])
+
+* With this release, the `NodeFilesystemAlmostOutOfSpace` no longer fires for certain read-only `tmpfs` instances. This change fixes an issue in which the alert fired for certain `tmpfs` mount points that were full by design. (link:https://issues.redhat.com/browse/OCPBUGS-6577[*OCPBUGS-6577*])
```

Comment:
There is nothing in the SSG or ISG specifically calling out the word "fire" except for "fire up" in the ISG https://www.ibm.com/docs/en/ibm-style?topic=word-usage#f I wonder if this is sort of the same thing and we should find another word to use here.

Reply:
How does "launch/es" sound? @opayne1

Reply:
I am good with that!